swarm/storage/schema.go

```diff
@@ -0,0 +1,6 @@
+package storage
+
+// "purity" is the first formal schema of LevelDB we release together with Swarm 0.3.5
```

Comment:
might be 0.4?

Reply:
next is 0.3.5 AFAIK, and it looks like we will be merging this before 0.3.5 is released.